### DISCUSSION TITLE
Error moving db files from old laragon 4 to laragon 6

### DISCUSSION THREAD

**Discussion by umsuka** [leokhoa/laragon#375] [General]
Hi All

I recently upgraded to laragon 6 and I had issues with dbs not showing up on db manager. 

I moved them to a new folder and reinstalled laragon then moved one db to test still not showing. I thought maybe there is some index missing and I tried to create the db via db manager while the old db file is copied to the new directory and I get this error.

```
SQL Error (3678): Schema directory '.\s******db' already exists. This must be resolved manually
 (e.g. by moving the schema directory to another location).
```

Any idea how can I keep old data and move it across to the new folder ?

Thanks

**Comment by Pen-y-Fan** [comment]
MySQL was changed from 5.7 to 8.0. The best way is to export using MySQL dump, **before upgrading,** and then import the required databases. Some system databases shouldn't be exported.
